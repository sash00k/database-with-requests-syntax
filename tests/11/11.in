get records where (department is "ММФ" or group is 19301)