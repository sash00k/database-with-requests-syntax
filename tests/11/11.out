Student Иванов С. В. from group 22129, ММФ
Student Петрова И. К. from group 19301, ФФ
Student Зуброва Д. А. from group 8126, ММФ
Student Иванова В. В. from group 22129, ММФ
Student Кузнецов И. П. from group 8126, ММФ